neurow/lib/neurow/public_api/endpoint.ex

```diff
@@ -20,7 +20,19 @@ defmodule Neurow.PublicApi.Endpoint do
   plug(:match)
   plug(:dispatch)
 
-  get "/v1/subscribe" do
+  match _ do
```

Comment:
It is not possible to configure Plug.router to use a dynamic prefix, or to pass a dynamic string to `get path do ... end`, so I ended using match to do the routing manually.

However, `use Plug.router` is still required to get the plug pipeline, to add `monitor_sse` and `Neurow.JwtAuthPlug` to the invocation chain.